extensions/eo/README.md

```diff
@@ -48,12 +48,28 @@ PlanetScope Ortho Tile Product has an `eo:gsd` of 3.7 (or 4 if rounding), even t
 3.125.   For example, one might choose for WorldView-2 the 
 Multispectral 20° off-nadir value of 2.07 and for WorldView-3 the Multispectral 20° off-nadir value of 1.38.
 
-**eo:constellation** is the name of the group of satellites that have similar payloads and have their orbits arranged in a way to increase the temporal resolution of acquisitions of data with similar geometric and radiometric characteristics. Examples are the Sentinel-2 [constellation](https://www.esa.int/Our_Activities/Observing_the_Earth/Copernicus/Sentinel-2/Satellite_constellation), which has S2A and S2B and RapidEye. This field allows users to search for Sentinel-2 data, for example, without needing to specify which specific platform the data came from.
-
+**eo:platform** is the unique name of the specific platform the instrument is attached to. For satellites this would 
+be the name of the satellite, whereas for drones this would be a unique name for the drone.  It is recommended that 
+the value be lowercase with no spaces. Examples include `landsat-8` (Landsat-8), `sentinel-2a` and `sentinel-2b` 
+(Sentinel-2), `terra` and `aqua` (part of NASA EOS, carrying the MODIS instruments),
+ `mycorp-uav-034` (hypothetical drone name), and `WORLDVIEW02` (Maxar/DigitalGlobe WorldView-2).
+ 
 **eo:instrument** is the name of the sensor used, although for Items which contain data from
 multiple sensors this could also name multiple sensors. For example, data from the Landsat-8
 platform is collected with the OLI sensor as well as the TIRS sensor, but the data is distributed
-together and commonly referred to as OLI_TIRS.
+together and commonly referred to as OLI_TIRS. Examples include `OLI_TIRS` (Landsat-8), `MSI` (Sentinel-2), 
+`ASTER` (Terra), and `MODIS` (Terra and Aqua).
```

Comment:
If instrument and platform should be lowercased, why are all instruments uppercased?